package/MDAnalysis/transformations/__init__.py

```diff
@@ -0,0 +1,60 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+"""\
+Trajectory transformations  --- :mod:`MDAnalysis.transformations`
+=================================================================
+
+The transformations submodule contains a collection of functions to modify the 
+trajectory. These transformation functions can be called by the user for any given
+timestep of the trajectory, added as a workflow using :meth:`add_transformations`
+of the :mod:`~MDAnalysis.coordinates.base` module, or upon Universe creation using
+the keyword argument `transformations`. Note that in the two latter cases, the workflow
+cannot be changed after being defined. Also, when adding a workflow as a keyword
+argument when defining the universe, note that some transformations may require 
+atomgroups.
+
+e.g. translate the coordinates of a given frame of a trajectory by a given vector:
+    
+    u = MDAnalysis.Universe(topology, trajectory)
+    new_ts = MDAnalysis.transformations.translate(1,1,1)(u.trajectory.ts)
+
+e.g. create a workflow and adding it to the trajectory:
+
+    u = MDAnalysis.Universe(topology, trajectory)
+    workflow = [MDAnalysis.transformations.translate(1,1,1), 
+                MDAnalysis.transformations.translate(1,2,3)]
+    u.trajetory.add_transformations(workflow)
+
+e.g. giving a workflow as a keyword argument when defining the universe:
+    
+    workflow = [MDAnalysis.transformations.translate(1,1,1), 
+                MDAnalysis.transformations.translate(1,2,3)]
+    u = MDAnalysis.Universe(topology, trajectory, transformations = workflow)
+    
```

Comment:
Here too it would make sense to explain what a transformation is expected to be. Latter on, when you will have written a bunch of these transformations, you'll be able to document different ways of writing a transformation.